### ISSUE TITLE
How to set chart settings?

### ISSUE THREAD

**Issue by brayancastrop** [Glavin001/ember-c3#45]
I have been looking through ember-c3 options but i have not found how to make use of c3 chart settings, for example
http://c3js.org/reference.html#oninit
http://c3js.org/reference.html#padding-left
and basically any option from the chart section, is there any way to make use of these options?

**Comment by maxwondercorn** [comment]
You use the component's padding property

```
   {{c3-chart` data=data title=title padding=padding}}
```

and then in a controller setup the padding

```
/* eslint ember/avoid-leaking-state-in-ember-objects: "off" */
export default Controller.extend({

   padding:  { left: 20 }

});
```

You will probably need the eslint rule at the top of the controller to eliminate linting errors caused by the object assignment in the controller. 

Not sure if you're also asking about the oninit event will be exposed in an upcoming version

**Comment by maxwondercorn** [comment]
See issue #77

**Comment by maxwondercorn** [comment]
oninit has been exposed in pr #79.  Update examples in dummy app show use with closure actions.  Update component will be published when issue #77 has been completed
